native/src/components/NavigationTiles.tsx

```diff
@@ -64,10 +65,12 @@ const NavigationTiles = ({ tiles }: NavigationTilesProps): ReactElement => {
   const scrollToStart = () => scrollViewRef.current?.scrollTo({ x: 0, y: 0, animated: true })
   const scrollToEnd = () => scrollViewRef.current?.scrollToEnd({ animated: true })
 
+  const { t } = useTranslation('common')
+
   return (
     <TilesRow>
       {isScrollable && (
-        <StyledPressable onPress={scrollToStart}>
+        <StyledPressable onPress={scrollToStart} accessibilityLabel={t('back')}>
```

Comment:
Sorry for the nitpicking here, but is back/forward really the right term here? I think previous/next would be more fitting.

Reply:
I think "forward" and "back" is better in this case because I would expect "previous" and "next" to be part of a pagination which we don't have here. But I'm not precious about it either 🤷

Reply:
Hmm. One more thought: Should these button even be visible to screenreaders? Aren't all items read to the user anyway? In that case it would just be confusing to announce the arrows at all. E.g. for Süddeutsche Zeitung and airbnb similar kind of arrows were just aria hidden.